src/OrchardCore/OrchardCore/Modules/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -272,6 +273,67 @@ private static void AddAntiForgery(OrchardCoreBuilder builder)
             });
         }
 
+        /// <summary>
+        /// Adds backwards compatibility to the handling of SameSite cookies.
+        /// </summary>
+        public static void AddSameSiteCookieBackwardsCompatibility(OrchardCoreBuilder builder)
+        {
+            builder.ConfigureServices(services =>
+                {
+                    services.Configure<CookiePolicyOptions>(options =>
+                    {
+                        options.MinimumSameSitePolicy = AspNetCore.Http.SameSiteMode.Unspecified;
+                        options.OnAppendCookie = cookieContext => CheckSameSiteBackwardsCompatiblity(cookieContext.Context, cookieContext.CookieOptions);
+                        options.OnDeleteCookie = cookieContext => CheckSameSiteBackwardsCompatiblity(cookieContext.Context, cookieContext.CookieOptions);
+                    });
+                });
```

Comment:
Do you have to call `app.UseCookiePolicy()` to enable the middleware?

Reply:
Yes, will add it.